### ISSUE TITLE
Cryptic error message when DMN is dropped on "Zeebe" Modeler

### ISSUE THREAD

**Issue by barmac** [camunda/camunda-modeler#2548]
__Describe the Bug__

Given I set `--disable-platform` flag, when I drop a DMN file on the modeler, it displays:

![image](https://user-images.githubusercontent.com/28307541/140274422-de4bca98-84d7-47cf-abb5-6630bb462a70.png)

__Steps to Reproduce__

See above.

__Expected Behavior__

It should say "... is not a BPMN or Form file."

__Environment__

- OS: [e.g. MacOS 10.2, Windows 10]
- Camunda Modeler Version: `develop`
- Execution Platform: [e.g. Camunda Platform, Camunda Cloud]
- Installed plug-ins: [...]

**Comment by pinussilvestrus** [comment]
Maybe related: https://github.com/camunda/camunda-modeler/pull/2217

**Comment by pinussilvestrus** [comment]
This fix has to direct to `develop` since we didn't release the `disable-platform` flag yet on `master`.

**Comment by Skaiir** [comment]
I don't believe this is related to #2214 

The way I understand it, the issue stems from the fact that the tabs providers (in TabsProvider.js) are not named: 

```js 
"cloud-bpmn": {
    name: null,
    encoding: ENCODING_UTF8,
    exports: {
      png: EXPORT_PNG,
```

I suspect this may have initially been done so that there wouldn't be a conflict when loading in files? Or as to not repeat the same formats in the error message? Anyways, the name of these providers are what that error message's file types are from.  I'll discuss with @pinussilvestrus for clarification on these providers since I'm not sure what else these names are used for.

**Comment by Skaiir** [comment]
I've made some modifications to the code and tests and everything looks to be working as expected. Here: [2832ec2](https://github.com/camunda/camunda-modeler/commit/2832ec2b623948932004619fd76de4cadf1a2e76)

The only question here is whether we are happy with the 'name' parameter, or if we should change it to something a little more descriptive. After inspection it looks to really only be used for describing the type of file a certain tabs provider handles.

I'm up for suggestions here, with @MaxTru we made mention of diagramType but this might not include stuff like forms (do they really count as diagrams). 

FileType, format, I'm not certain really. Perhaps just leave it as is? @pinussilvestrus
